app/src/main/java/org/greenstand/android/TreeTracker/walletselect/WalletSelectViewModel.kt

```diff
@@ -32,10 +31,8 @@ class WalletSelectViewModel(private val userRepo: UserRepo) : ViewModel() {
                     _state.value = _state.value?.copy(
                         currentUser = currentUser,
```

Comment:
Now that it's Flow, I don't think we need the '?.' anymore

Reply:
okay,
Going to replace this with a dot call